opencti-platform/opencti-graphql/src/schema/general.js

```diff
@@ -98,3 +99,27 @@ export const ABSTRACT_TYPES = [
 ];
 export const isAbstract = (type) => R.includes(type, ABSTRACT_TYPES);
 // region utils
+
+// Knowledge
+export const KNOWLEDGE_TYPES = [
```

Comment:
Can't we deduce this type from already existing types?
Like SDO/SCO/SMO/SCO

Or not internal type ?

I am not really comfortable with adding a type relative to front-end

Reply:
the const KNOWLEDGE_TYPES was already existing. I just added ENTITY_TYPE_CONTAINER_CASE in it